db/migrate/20221201105921_addnewtablestocahe.rb

```diff
@@ -3,6 +3,6 @@ def change
     add_column :caches, :title, :string
     add_column :caches, :user_id, :integer
     # add_index :caches, :user_id
-    add_column :caches, :timestamps
+    # add_column :caches, :timestamps
```

Comment:
Kindly add just :datetime because rails were expected to get 3 elements as an argument.